components/dashboard/src/projects/NewProject.tsx

```diff
@@ -327,15 +327,17 @@ export default function NewProject() {
                                 <div key={`repo-${index}-${r.account}-${r.name}`} className="flex p-3 rounded-xl hover:bg-gray-100 dark:hover:bg-gray-800 focus:bg-gitpod-kumquat-light transition ease-in-out group" >
 
                                     <div className="flex-grow">
-                                        <div className="text-base text-gray-900 dark:text-gray-50 font-medium rounded-xl whitespace-nowrap">{toSimpleName(r.name)}</div>
+                                        <div className={"text-base text-gray-900 dark:text-gray-50 font-medium rounded-xl whitespace-nowrap" + (r.inUse ? " text-gray-400" : "text-gray-700")}>{toSimpleName(r.name)}</div>
                                         <p>Updated {moment(r.updatedAt).fromNow()}</p>
                                     </div>
                                     <div className="flex justify-end">
-                                        <div className="h-full my-auto flex self-center opacity-0 group-hover:opacity-100">
+                                        <div className="h-full my-auto flex self-center opacity-0 group-hover:opacity-100 items-center mr-2 text-right">
                                             {!r.inUse ? (
                                                 <button className="primary" onClick={() => setSelectedRepo(r)}>Select</button>
                                             ) : (
-                                                <p className="my-auto">already taken</p>
+                                                <p className="text-gray-500 font-medium">
+                                                    @{r.inUse.userName} already<br/>added this repo
```

Comment:
**issue**: Currently, if you add a project and remove it you get an error when you try to add it again and breaks the product with no visible action forward. Known? ❗ 

| Error | Empty Page |
|-|-|
|  <img width="488" alt="Screenshot 2021-12-13 at 10 59 00 PM" src="https://user-images.githubusercontent.com/120486/145887929-eea15ecc-7878-4492-81e2-a3dbaf6e9f27.png"> | <img width="1083" alt="Screenshot 2021-12-13 at 10 59 59 PM" src="https://user-images.githubusercontent.com/120486/145887988-c42c2682-8ab0-45e7-8738-cb43ec85a86e.png"> |

Reply:
@gtsiolis Just tested this while reviewing together with Alex and it worked well. Is this still happening for you?

Reply:
@laushinka It seems to work ok now. 🏓